doc/source/reference/development/howtoadd.rst

```diff
@@ -11,12 +11,19 @@ Python
 Python (and Cython) code in **freud** should follow `PEP 8 <https://www.python.org/dev/peps/pep-0008/>`_.
 
 During continuous integration (CI), all Python and Cython code in **freud** is tested with `flake8 <http://flake8.pycqa.org/>`_ to ensure PEP 8 compliance.
-It is strongly recommended to `set up a pre-commit hook <http://flake8.pycqa.org/en/latest/user/using-hooks.html>`_ to ensure code is compliant before pushing to the repository:
+Additionally, all CMake code is tested using `cmakelang <https://cmake-format.readthedocs.io/en/latest/index.html>`__.
+It is strongly recommended to `set up a pre-commit hook <https://pre-commit.com/>`_ to ensure code is compliant before pushing to the repository:
 
 .. code-block:: bash
 
-    flake8 --install-hook git
-    git config --bool flake8.strict true
+    pip install -r requirements-precommit.txt
+    pre-commit install
+
+To manually run `Pre-commit`_ for all the files present in the repository, run the following command:
```

Comment:
Does this link somewhere?

Reply:
Nope: 
![image](https://user-images.githubusercontent.com/3943761/98051397-698f6700-1df9-11eb-8104-f7942e6bd388.png)

Reply:
Good catch, I copied this particular note from the signac-flow documentation and forgot to replace the link.